### ISSUE TITLE
[Bug] TypeError: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by genofire** [ffrgb/meshviewer#11]
https://github.com/ffrgb/meshviewer/blob/bcd126464f505fa61d2390394ff1642761d734e2/lib/map.js#L278

In der `config.json` wird bei der Map/Tiles eine Zeitangabe benötigt, mehrere Optionen:
* villt. sollt dies nicht required sein
* Dokumentieren und drauf hinweisen + für Travis Test schreiben

```JSON
{
  "DON'T CHANGE!": "{{ ansible_managed }}",
  "dataPath": "{{ meshviewer_web_path }}",
  "siteName": "Freifunk Bremen",
  "mapSigmaScale": 0.5,
  "showContact": false,
  "geoJson":true,
  "maxAge": 3,
  "disabledNodeAlert": true,
  "globalInfos": [
    {
      "name": "Wochenstatistik",
      "href": "https://grafana.bremen.freifunk.net/dashboard/db/globals",
      "thumbnail": "https://grafana.bremen.freifunk.net/render/dashboard-solo/db/globals?pa$
      "caption": "Entwicklung der Anzahl der Knoten und der Clients innerhalb der letzten 7$
    }
  ],
  "nodeInfos": [
    {
      "name": "Clientstatistik",
      "href": "https://grafana.bremen.freifunk.net/dashboard/db/global-selectable?var-node=$
      "thumbnail": "https://grafana.bremen.freifunk.net/render/dashboard-solo/db/global-sel$
      "caption": "Entwicklung der Anzahl der Clients innerhalb der letzten 7 Tage"
    }
  ],
  "mapLayers": [
    {
      "name": "Freifunk Bremen",
      "url": "//tiles.bremen.freifunk.net/{z}/{x}/{y}.png",
      "config": {
        "type": "osm",
        "attribution": "<a href=\"https://github.com/mapbox/mapbox-studio-osm-bright.tm2\">$
        "maxZoom": 20
      }
    },
    { "name": "OpenStreetMap.Mapnik" },
    { "name": "Stamen.TonerLite" }
  ]
}
```

**Comment by xf-** [comment]
Which browser? Can't reproduce the behavior in Firefox and Chrom(e|ium).

Leaftlet.Providers is not included, copy full definition. Overhead and easy mistakes like http provider on https site.
```
    { "name": "OpenStreetMap.Mapnik" },
    { "name": "Stamen.TonerLite" } 
```

**Comment by xf-** [comment]
https://github.com/ffrgb/meshviewer/commit/89b9a4216e887b52f9d952da4311a3f31678f88b added documentation for the additional arguments.

**Comment by genofire** [comment]
Better? #12

**Comment by xf-** [comment]
i also try to improve the code, to avoid the issue ;)

Currently testing:
` if ((typeof item.config.start === 'number' && item.config.start <= now.getHours()) || (typeof item.config.end === 'number' && item.config.end > now.getHours())) {`

**Comment by xf-** [comment]
solved via f3eb100
